en/docs/getting-started/about-this-release.md

```diff
@@ -2,113 +2,108 @@
 
-- Widget generator tool
+* Widget generator tool
 
-     In addition to the default widgets that are shipped with the analytics dashboards, the Widget generator tool makes it easier to implement your own custom widget and use them across the dashboards. The widget generator tool automatically generates the widget skeleton by taking into account the user preferences via a command-line tool and guides the user until the widget is deployed in the dashboard.
+    In addition to the default widgets that are shipped with the analytics dashboards, the Widget generator tool makes it easier to implement your own custom widget and use them across the dashboards. The widget generator tool automatically generates the widget skeleton by taking into account the user preferences via a command-line tool and guides the user until the widget is deployed in the dashboard.
 
-- GraphQL Analytics
-    
-     GraphQL Analytics requirements differ from standard API analytics. The existing charts have been improved to be able to view statistics for GraphQL operations. A single API call for a GraphQL API can have multiple operations associated with it. In this release, analytics charts will have the in-built capability to filter these API calls by GraphQL operation. 
+* GraphQL Analytics
 
+    GraphQL Analytics requirements differ from standard API analytics. The existing charts have been improved to be able to view statistics for GraphQL operations. A single API call for a GraphQL API can have multiple operations associated with it. In this release, analytics charts will have the in-built capability to filter these API calls by GraphQL operation.
 
 ## What has changed in this release
 
 ### Improvements to existing WSO2 API-M features
 
-- [Language Switch]({{base_path}}/learn/consume-api/customizations/adding-internationalization/#enabling-the-language-switch)
-    
+* [Language Switch]({{base_path}}/develop/customizations/adding-internationalization/#enabling-the-language-switch)
+
     A language switch allows the Developer Portal users to select the language in which they want to read the content. The Developer Portal theming architecture lets developers to add a language switch in different ways.
 
-- [Changing the layout direction in the Developer Portal]({{base_path}}/learn/consume-api/customizations/adding-internationalization/#changing-the-layout-direction)
+* [Changing the layout direction in the Developer Portal]({{base_path}}/develop/customizations/adding-internationalization/#changing-the-direction-of-the-ui)
 
     You can switch the Developer Portal layout direction from its default LTR to RTL using a single configuration.
 
-- Custom URL feature improvements
-
-- Tracing feature improvements
-
-- Multiple JWT issuer support
+* Custom URL feature improvements
+* Tracing feature improvements
+* Multiple JWT issuer support
+* Custom data publishing via a sequence
```

Comment:
why have we changed?

Reply:
The same heading was repeated twice `Improvements to existing WSO2 API-M features`. Hence, removed the second and made it under the same roof.